addon/ng2/blueprints/route-config/index.js

```diff
@@ -1,13 +1,17 @@
 var fs = require('fs-extra');
 var path = require('path');
+var dynamicPathParser = require('../../utilities/dynamic-path-parser');
 
 var imports, routeDefinitions;
 
 module.exports = {
   description: 'Registers the route with the router.',
 
-  locals: function (options) {
-    return generateLocals.call(this, options);
+  normalizeEntityName: function () {
+    var parsedPath = dynamicPathParser(this.project, 'ignore');
```

Comment:
Yes, what I've been adding is to put the underlining support in to allow the blueprints to align with the style guide